robot/impl/local_robot_test.go

```diff
@@ -1893,24 +1894,15 @@ func TestConfigPackageReferenceReplacement(t *testing.T) {
 		},
 		Services: []config.Service{
 			{
-				Name: "Vision-Service",
-				Type: vision.SubtypeName,
-				ConvertedAttributes: &vision.Attributes{
-					ModelRegistry: []vision.VisModelConfig{
-						{
-							Type: "tflite_classifier",
-							Name: "my_classifier",
-							Parameters: config.AttributeMap{
-								"model_path":  "${packages.package-1}/model.tflite",
-								"label_path":  "${packages.package-2}/labels.txt",
-								"num_threads": 1,
-							},
-						},
-					},
+				Name: "ml_model",
+				Type: "tflite_detector",
+				ConvertedAttributes: &tflitecpu.TFLiteConfig{
+					ModelPath:  "${packages.package-1}/model.tflite",
+					LabelPath:  &labelPath,
+					NumThreads: 1,
 				},
 			},
```

Comment:
rather than using the vision service, could you the ml model service? And could you test on this specific  making service config:

```
PackagePath: packageDir,
Services: []config.Service{
  {
    Name: "my_ml_model_service",
    Type: mlmodel.SubType,
    Model: resource.NewDefaultModel("tflite_cpu"),
    Attributes: config.AttributeMap{
      "model_path": ${packages.package-1}/model.tflite"
      "label_path": ${packages.package-1}/labels.txt"
      "num_threads":  1
    }
  }
}
```

Reply:
We would like to see if the RobotConfig can successfully parse and use the Walk function that you created for the tflite model